src/components/landingPage/listOfArticles/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import ArticleCard from '../../common/articleCard';
+import * as actionTypes from '../../../redux/actions/actionTypes';
+
+export default function ListOfArticles({ articles, limitArticlesNumber }) {
+  return articles.map((article, index) => {
+    if (index >= limitArticlesNumber) return false;
+
+    return (
+      <a
+        key={article && article.slug}
+        href={`${actionTypes.BACKEND_URL}/api/${actionTypes.VERSION}/articles/${article
+          && article.slug}`}
```

Comment:
It should not go to the back end. for now, let it go to ```/articles/slug``` then the person implementing view single article will add the path to routes